cubecrit/controllers/manufacturers.py

```diff
@@ -21,7 +23,21 @@ def get_manufacturer_route(external_id: str) -> str:
     Returns an HTML page.
     """
     with current_app.config["db"].connect() as connection:
+        num_pages = Puzzle.get_num_pages(connection)
```

Comment:
`Puzzle.get_num_pages` needs to be able to accept a `Manufacturer` to filter out rows when counting, otherwise it's going to return too large of a page count.